tests/console/zdup_prepare_leap_sle.pm

```diff
@@ -0,0 +1,104 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: prepare migration from Leap to SLE
+#   Documentation for 15:
+#   https://www.suse.com/de-de/documentation/sles-15/book_sle_upgrade/data/sec_upgrade-online_opensuse_to_sle.html
+#   12 is not supported and inoffical
+# Maintainer: lemon.li <leli@suse.com>
+
+use base "console_yasttest";
```

Comment:
Hmm, I wonder if you use this base class, might be that `installbasetest` will be a better fit here or maybe `y2_installbase` in case you need yast related post fail hooks. Then you can skip adding fatal flag here.

Reply:
Replaced it with installbasetest, but need use 'registration'.